### PR TITLE
Update HackHands to 1.3.2.76

### PR DESCRIPTION
**Pull request by sergiopvilar**

### CODE REVIEW

File: Casks/hackhands.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask :v1 => 'hackhands' do
-  version '0.1.1'
-  sha256 'f3d5b67a79d963f88ddc03d2e5947a19a616a670442b2b1389159bd3960dcd8d'
+  version '1.3.2.76'
+  sha256 :no_check
```
Comment (alebcay) [inline Casks/hackhands.rb]: Because you are using a versioned `url`, you must provide a `sha256`.

Comment (sergiovilar) [inline Casks/hackhands.rb]: Oh, okay! I'll fix. Thank you.

**Comment by sergiovilar** [comment]
@alebcay Fixed, please take a look.

**Comment by vitorgalvao** [comment]
You’ve done the reverse. @alebcay mentioned adding a `sha256`, not unversioning the cask altogether.

**Comment by vitorgalvao** [comment]
Thank you for the submission, @sergiovilar. Will fix in a different commit.

**Comment by sergiovilar** [comment]
@vitorgalvao That's because we have constantly updates, a link do the latest version is more viable to us. Could you please reopen?

**Comment by vitorgalvao** [comment]
We prefer versioned URLs, as among other things they provide greater security for users, and that takes precedence. It is also the url linked on the website, which means sooner or later someone will notice and submit a fix anyway. You’d also be breaking update functionality, for when it lands.

If this is a case of you wanting to provide a `latest` download via homebrew-cask to your users, then you should host the cask somewhere and tell users to run `brew cask install <url_to_your_custom_cask>`. The version in our official repo, however, must follow the same rules as every other cask.

**Comment by sergiovilar** [comment]
I see, but the documentation says that I could use the `:latest` parameter. Why document a thing that could be not approved? I can't understand.

I made a quick search in the repository and I found 926 ocurrences of `:latest`, why I can't use?

**Comment by vitorgalvao** [comment]
Because in those cases, it isn’t a choice. Cases that use `:latest` do not have a versioned download. That is not the case here.

**Comment by sergiovilar** [comment]
Okay, but if I have a weekly release, I need to make a PR in each week for this repo?

**Comment by vitorgalvao** [comment]
Yes, if it is to be kept up to date. If you control the release that’s easier, though, the submission could be automated.

**Comment by sergiovilar** [comment]
Okay, thank you. I'll find a way to automate the submission.

**Pull request closed without merging**